<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>connectForm</class>
 <widget class="QDialog" name="connectForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>186</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки подключения</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelLogin">
     <property name="text">
      <string>Логин:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEditLogin">
     <property name="placeholderText">
      <string>Логин</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelPassw">
     <property name="text">
      <string>Пароль:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEditPassw">
     <property name="placeholderText">
      <string>Пароль</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelHost">
     <property name="text">
      <string>Хост:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEditHost">
     <property name="placeholderText">
      <string>Хост</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelDbName">
     <property name="text">
      <string>БД:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEditDbName">
     <property name="placeholderText">
      <string>База данных</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="checkBoxIsAutoConnect">
     <property name="text">
      <string>Авто подключение</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
